README.md

```diff
@@ -87,12 +87,14 @@ configuration, ipamD always try to keep one extra ENI.
 When number of pods running on the node exceeds the number of addresses on a single ENI, the CNI backend start allocating
 a new ENI and start using following allocation scheme:
 
-For example, a m4.4xlarge node can have up to 8 ENIs, and each ENI can have up to 30 IP addresses.
-(https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html).
+For example, a m4.4xlarge node can have up to 8 ENIs, and each ENI can have up to 30 IP addresses. See 
+[Elastic Network Interfaces documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html) for details.
 
-* If the number of current running Pods is between 0 to 29, ipamD will allocate one more eni. And Warm-Pool size is 2 eni * (30 -1) = 58
+* If the number of current running Pods is between 0 and 29, ipamD will allocate one more eni. And Warm-Pool size is 2 eni * (30 -1) = 58
 * If the number of current running Pods is between 30 and 58, ipamD will allocate 2 more eni. And Warm-Pool size is 3 eni * (30 -1) = 87
 
+For a detailed explanation, see [`WARM_ENI_TARGET`, `WARM_IP_TARGET` and `MINIMUM_IP_TARGET`](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/eni-and-ip-target.md).
```

Comment:
Seems the link (https://github.com/aws/amazon-vpc-cni-k8s/blob/master/docs/eni-and-ip-target.md) has been removed (404). Please check it.

Reply:
It's the other way around, I'm adding it in this PR. 😄